pallets/offchain-worker/src/lib.rs

```diff
@@ -101,9 +101,11 @@ pub mod pallet {
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		fn on_initialize(block_number: T::BlockNumber) -> Weight {
 			log::info!("ocw on_initialize {:?}.", block_number);
+// SBP M1: magic value?
```

Comment:
it is just a place holder, not magic value. we are struggling about put the aggregation method in on_initialize or on_finalize. what's difference, if some forked block, different condition in isolated chain or parachain, collator node and non-collator node. could @jeluard give some advice on it.

Reply:
`on_finalize` weight has to be considered in `on_initialize`

Reply:
understood, thanks